### PR TITLE
feature/i contactUs interface

### PR DESCRIPTION
**Pull request by ThaminduChankana**

### CODE REVIEW

File: src/interfaces/IContactUs.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  email: string;
+  message: string;
+  createdAt: Date;
+  updatedAt: Date;
+  deletedAt?: Date | null;
+}
+
+//Contact Us State interface
+interface IContactUsStore {
+  contact: IContactUs | null;
```
Comment (miyurugunarathna) [inline src/interfaces/IContactUs.ts:14]: Change contact to contactUs and do the same for others.

File: src/interfaces/IContactUs.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  name: string;
+  email: string;
+  message: string;
+  createdAt: Date;
+  updatedAt: Date;
+  deletedAt?: Date | null;
+}
+
+//Contact Us State interface
+interface IContactUsStore {
```
Comment (miyurugunarathna) [inline src/interfaces/IContactUs.ts:13]: Add selectedContactUsId and addContactUs too

File: src/interfaces/IContactUs.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  contact: IContactUs | null;
+  contacts: IContactUs[] | null;
+  ArchiveContact: IContactUs | null;
```
Comment (miyurugunarathna) [inline src/interfaces/IContactUs.ts:16]: Change archive to deleted and do the same for other one. Also make sure to use simple letter as first letter.

**Review by miyurugunarathna — CHANGES_REQUESTED** [review]

File: src/interfaces/IContactUs.ts (hunk 2 above)
Comment (senuravihanjayadeva) [reply src/interfaces/IContactUs.ts:13]: addContactUs is not required in admin panel @miyurugunarathna @ThaminduChankana 
It is handling in our web

**Comment by senuravihanjayadeva** [comment]
And Please change the PR name also

File: src/interfaces/IContactUs.ts (hunk 2 above)
Comment (miyurugunarathna) [reply src/interfaces/IContactUs.ts:13]: Ahh yes yes.

**Review by rusiruavb — APPROVED** [review]
LGTM

**Review by miyurugunarathna — APPROVED** [review]
LGTM

**Comment by senuravihanjayadeva** [comment]
Please use a Proper name for PR

**Pull request closed without merging**